### PR TITLE
feat: [arco] add vue site and language config

### PR DESCRIPTION
**Pull request by yuhan0709**
# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### What is the expected behaviour?

I want to add two configurations:
1. Distinguish the usage documentation of `react` and `vue`.
2. Distinguish between `zh-CN` and `en-US` document languages

##### NB2: Any other feedback / questions ?
I changed `start_urls`, but these two URLs have overlapping scenes, I'm not sure if this is possible.

If not, is there any good solution?

Looking forward to your reply， THX.

### CODE REVIEW

File: configs/arco.json [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+      "selectors_key": "vue"
+    },
+    {
+      "url": "https://arco.design/react/*",
+      "tags": ["react"]
+    },
+    {
+      "url": "https://arco.design.+?(en-US)",
+      "tags": ["en-US"]
+    }
```
Comment (shortcuts) [inline configs/arco.json:16]: Matching patterns also work for `start_urls` so you don't need wildcards or variables when there's only one option

Comment (yuhan0709) [reply configs/arco.json]: I have changed it, but in the case of en-US, we still need to use regular to match

**Comment by yuhan0709** [comment]
@shortcuts hello, is this pr still being checked?

**Review by shortcuts — APPROVED** [review]

**Comment by yuhan0709** [comment]
I want to know how long it will take effect after the modification? I currently see that I still can’t find what I want.

**Comment by yuhan0709** [comment]
I want to manually trigger the crawler update but why can't log in to the crawler :( 
image

**Comment by yuhan0709** [comment]
I login in the crawlers, But it’s empty, I can’t manually trigger the crawler :(
image

**Comment by shortcuts** [comment]
> I want to know how long it will take effect after the modification? I currently see that I still can’t find what I want.

Hey, crawls are triggered after a merge, so you should already see the changes

> I login in the crawlers, But it’s empty, I can’t manually trigger the crawler :(

Your website has not been migrated to the Crawler yet, see https://docsearch.algolia.com/docs/migrating-from-legacy

**Comment by yuhan0709** [comment]
> Hey, crawls are triggered after a merge, so you should already see the changes
> 
I have tagged different startUrls, but I cannot find the content even after adding the searchParameters, Is there a way I can see the captured data.

> Your website has not been migrated to the Crawler yet, see https://docsearch.algolia.com/docs/migrating-from-legacy

For this migration, maybe I should receive a migration email?  Looking forward :)

**Comment by shortcuts** [comment]
> I have tagged different startUrls, but I cannot find the content even after adding the searchParameters, Is there a way I can see the captured data.

Sorry we don't have a way to provide access to DocSearch users yet, if URLs/records are missing the best bet would be to add a `sitemap.xml` to your website and/or include client-side rendering options if necessary: https://docsearch.algolia.com/docs/legacy/config-file#js_render-optional

> For this migration, maybe I should receive a migration email? Looking forward :)

Yep exactly, you'll receive one once the migration is done for your index!

**Comment by yuhan0709** [comment]
I’m here again~ I really don’t know why I added `facetFilters: ['tags:en-US'],` but I can’t find anything...😭
This is our website https://arco.design/react/en-US/docs/start

**Comment by shortcuts** [comment]
`react` will be picked before `en-US` https://github.com/algolia/docsearch-configs/pull/4865/files#diff-19df1e2cd29582f371fddacc3cd6eab952f1ffb228324bca255da1c7ba6c09b6R10

`start_urls` are picked hierarchically

**Comment by yuhan0709** [comment]
So if my path is `/react/en-US/docs/start`, it actually hits the `tag` of `react`? And why is there also nothing when I search in `tags:vue`. 👉https://arco.design/vue/en-US/docs/start

**Pull request merged**